nginx.conf.sample

```diff
@@ -177,6 +177,10 @@ location /errors/ {
     }
 }
 
+location /vendor/ {
```

Comment:
Why do you have `vendor` directory inside `pub`?

Reply:
Oh, i didnt realize the pubfolder is used as doc root - sorry!